Blog: The best Plausible alternatives (#9543)
PostHog/posthog.com: contents/blog/best-plausible-alternatives.mdx

## 2. Fathom

- **Founded:** 2018
- **Similar to:** Plausible, Google Analytics
- **Typical users:** Founders and content teams
- **Typical customers:** Privacy-conscious, content-focused websites

![Fathom](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_10_02_at_16_12_06_c41b89ea2f.png)

### What is Fathom?

Fathom is a privacy-focused [Google Analytics alternative](/blog/ga4-alternatives). It is a simple web analytics tool that captures details on traffic, sessions, referrers, sources, and campaigns. It does this while helping users stay compliant and avoid cookies.

### Key features

- 🧑‍🎨 **Simple web analytics:** Fathom provides a simple, GA-like experience to monitor your website and get an overview of your traffic.

- 🍪 **Cookieless:** Avoid cookie banners and track your site without using cookies.

- 📊 **Marketing metrics:** Track the sources, devices, browsers, locations, UTMs, and more for your traffic. 

- ⭐ **Conversions:** Track custom events and analyze them as conversions towards a goal.

### How does Fathom compare to Plausible?

> Feature-wise, Plausible and Fathom are nearly identical and this shows on their website and design. Looking closer, Fathom is missing some of the compliance features Plausible offers, like [open source](/blog/best-open-source-analytics-tools) code and EU hosting.

Reviewer: ```suggestion
Feature-wise, Plausible and Fathom are nearly identical. Looking closer, however, Fathom is missing some of the compliance features Plausible offers, like [open source](/blog/best-open-source-analytics-tools) code and EU hosting.
```

Reviewer: We might need to explain what Fathom does instead of EU-hosting here. iirc, it was some kind of routing / residency thing going on, but I can't recall the specifics.

  <ComparisonRow column1="Events only" column2={true} feature="Custom events" description="Manually capture custom events and properties wherever they happen" />
  <ComparisonRow column1={false} column2={false} feature="User profiles" description="Capture and analyze data at the user-level" />
  <ComparisonRow column1={false} column2={false} feature="Cohort analysis" description="Combine users into cohorts for analysis" />
  <ComparisonRow column1={false} column2={false} feature="Session replay" description="Watch real users on your site; discover friction points" />
  <ComparisonRow column1={false} column2={true} feature="Reverse proxy" description="Official instructions to capture more accurate data by avoiding tracking blockers" />
</ComparisonTable>

### How popular is Fathom?

According to [BuiltWith](https://trends.builtwith.com/analytics/Fathom-Analytics), as of October 2024, 1,832 of the top 1 million websites deploy Fathom. This is just under half of Plausible's 3,500.

### Why do companies use Fathom?

Fathom has limited reviews on G2, but much more praise on social. Judging from social posts, the main reasons people choose Fathom are:

> 1. **Privacy-focus:** Fathom reviewers almost always praise their focus on privacy. This allows users to stay compliant with regulations like GDPR, CCPA, and more.

Reviewer: ```suggestion
1. **Privacy-focus:** Fathom reviewers almost always praise its focus on privacy. This allows users to stay compliant with regulations like GDPR, CCPA, and more.
```

  <ComparisonRow column1={true} column2={true} feature="Reverse proxy" description="Official instructions to capture more accurate data by avoiding tracking blockers" />
</ComparisonTable>

### How popular is Piwik Pro?

According to [BuiltWith](https://trends.builtwith.com/analytics/Piwik-PRO), as of October 2024, 1,902 of the top 1 million websites deploy Piwik Pro. 3,500 use Plausible.

### Why do companies use Piwik Pro?

Based on reviews from [G2](https://www.g2.com/products/piwik-pro/reviews), the following summarizes of reasons why users choose Piwik Pro:

1. **GA-like:** Google Analytics users find that the transition to Piwik Pro is an easy one thanks to their similar dashboards and data models.

2. **Compliance:** Those in strict regulatory environments appreciate how Piwik Pro provides many of the tools they need to stay compliant like tag and consent management.

3. **Control:** Reviewers like that they are in control of their data, tune it to their needs, and host it outside the US.

> #### Bottom line
> For users looking for the compliance features Piwik Pro offers, it's a good alternative to Plausible, especially because of Piwik Pro's free tier and privacy focus. 

> ## 6. LogRocket

Reviewer: Does LogRocket make sense here? Feels like a weird choice. Umami would make more sense, imo.
